scripts/globals/items/flask_of_primeval_brew.lua

```diff
@@ -0,0 +1,98 @@
+-----------------------------------------
+-- ID: 5853
+-- Item: Flask of Primeval Brew
+-- Status Effect: 3 minutes, All Races
+-----------------------------------------
+-- HP +9000
+-- MP +9000
+-- Strength +900
+-- Dexterity +900
+-- Vitality +900
+-- Agility +900
+-- Intelligence +900
+-- Mind +900
+-- Charisma +900
+-- Attack +9000
+-- Defense +9000
+-- Accuracy +1000
+-- Evasion +1000
+-- Magic Attack Bonus +900
+-- Regen +300
+-- Refresh +300
+-- Regain +50
+-- Ranged Accuracy +1000
+-- Ranged Attack +9000
+
+require("scripts/globals/status");
+
+-----------------------------------------
+-- OnItemCheck
+-----------------------------------------
+
+function onItemCheck(target)
+local result = 0;
```

Comment:
Your indentation and formatting are wrong. Also `== true` isn't needed to check if its true.
```lua
function onItemCheck(target)
    local result = 0;
    if (target:hasStatusEffect(EFFECT_TRANSCENDENCY)) then
        result = 523;
    end
    return result;
end;
```
Its also possible to just else the result if you want (doesn't really matter though, do as you see fit) instead of variablizing it.
```lua
function onItemCheck(target)
    if (target:hasStatusEffect(EFFECT_TRANSCENDENCY)) then
        return 523;
    else
        return 0;
    end
end;
```

The trunk currently has horrible inconsistency all over because many cooks one pot + older code wasn't held to very good (or any?) standards so picking an existing script to model ones work on is a crapshoot.